code/datums/components/crafting/guncrafting.dm

```diff
@@ -19,3 +22,49 @@
 	desc = "A long piece of silk with some resemblance to cable coil."
 	icon = 'icons/obj/improvised.dmi'
 	icon_state = "silkstring"
+
+////////////////////////////////
+// KAT IMPROVISED WEAPON PARTS//
+////////////////////////////////
+
+/obj/item/weaponcrafting/improvised_parts
+	name = "Nuclear Syndicate Hyper Giga-Operative Undercover 5th-Dimensional Super-Tank"
+	desc = "An undercover hypno sissification operative attempting to make Nanotrasen homosexual. Report this to Central immediately. Something is off here."
```

Comment:
Please Don't do this. Just give it a normal name.

Reply:
Also you do not need to add your name to everything you code in the code. I've seen this regularly from you in PRs and you should knock it off.

Reply:
This is not resolved. Change it to a normal name and description.

Reply:
This isnt resolved just because you hid the comment.